lib/content/eslintrc-js.hbs

```diff
@@ -28,5 +24,8 @@ module.exports = {
   extends: [
     '@npmcli',
     ...localConfigs,
+    {{#if prettier}}
```

Comment:
@lukekarrys @jumoel Is there a reason we're making `prettier` usage configurable? Are there repos we know we don't want to use `prettier` in? Do we want a gradual or coordinated rollout of `prettier`?

Reply:
As I recall, it took some semi-manual effort to roll out prettier to a new repo due to the `.git-blame-ignore-revs` file and getting the right SHA after merging the PR. So it was made configurable with the goal of turning it on everywhere and then changing the default after that.